src/components/Tooltip/Tooltip.spec.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import Tooltip from '.';
+
+describe('Tooltip', () => {
+  it('aligns to the center', () => {
```

Comment:
I phrased it like this, too 😄 But we agreed to use `should...` in the team.

Reply:
Ops. habits :) 
Let me change